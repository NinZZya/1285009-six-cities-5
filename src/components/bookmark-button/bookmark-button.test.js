import React from 'react';
import renderer from 'react-test-renderer';
import BookmarkButton from './bookmark-button';


it(`Should BookmarkButton render correctly`, () => {
  const tree = renderer
    .create(
        <BookmarkButton />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
