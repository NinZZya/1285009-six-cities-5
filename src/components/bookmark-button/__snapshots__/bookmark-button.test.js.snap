// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should BookmarkButton render correctly 1`] = `
<button
  className="undefined__bookmark-button button "
  type="button"
>
  <svg
    className="undefined__bookmark-icon"
    height={19}
    width={18}
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>
  <span
    className="visually-hidden"
  >
    To bookmarks
  </span>
</button>
`;
